cves/CVE-2013-3674.yml

```diff
@@ -90,19 +94,16 @@ fixes_vcc_instructions: |
   find interesting in there.
 fixes:
 - commit: ad002e1a13a8df934bd6cb2c84175a4780ab8942
-  note: ''
+  note: 'The inital fix for the bug, a simple buffer size check is added'
```

Comment:
I like how you have two commits that fixes the issue, but maybe the second one can be moved to interesting commits?

Reply:
```suggestion
  note: 'The inital fix for the vulnerability, where a simple buffer size check is added.'
```

Reply:
The second is still a fix. Interesting commits should have happened between VCC and the fixes which the 2nd fix commit does not meet.